src/mixins/themeable.ts

```diff
@@ -74,25 +74,16 @@ export interface ThemeableMixin extends Evented {
  * Themeable
  */
 export interface Themeable extends ThemeableMixin {
-	baseClasses: BaseClasses;
+	baseClasses: {};
```

Comment:
Nope, if I make it `BaseClasses` you lose the typings on the css file you import because you have to declare it as `BaseClasses` which is basically just a string index object used within this file to make TS happy.